index.html

```diff
@@ -4778,41 +4789,28 @@ <h2 id="AudioBufferSourceNode">
           </dt>
           <dd>
             <p>
-              An optional value in seconds where looping should begin if the
-              <code>loop</code> attribute is true. Its default
+              An optional <a>playhead position</a> where looping should begin
```

Comment:
I didn't see anything wrong with the original text and don't think it contradicts in any way the new text.  I don't find the playhead position adding any additional info.  WDYT?